File: app/src/main/java/com/intfocus/yonghuitest/bean/kpi/msg/MeterRequestResult.java
package com.intfocus.yonghuitest.bean.kpi.msg;


import com.intfocus.yonghuitest.bean.kpi.MererEntity;

import java.util.ArrayList;

/**
 * 仪表数据请求结果
 * Created by zbaoliang on 17-4-28.
 */
public class MeterRequestResult {
    public boolean isSuccress ;
    public int stateCode ;
    public ArrayList<MererEntity> topDatas;
    public ArrayList<MererEntity> bodyDatas;

    public MeterRequestResult(boolean isSuccress, int stateCode) {
        this.isSuccress = isSuccress;
        this.stateCode = stateCode;
    }

    public boolean isSuccress() {
        return isSuccress;
    }

    public void setSuccress(boolean succress) {
        isSuccress = succress;
    }

    public int getStateCode() {
        return stateCode;
    }

    public void setStateCode(int stateCode) {
        this.stateCode = stateCode;
    }

    public ArrayList<MererEntity> getTopDatas() {
        return topDatas;
    }

    public void setTopDatas(ArrayList<MererEntity> topDatas) {
        this.topDatas = topDatas;
    }

    public ArrayList<MererEntity> getBodyDatas() {
        return bodyDatas;
    }

    public void setBodyDatas(ArrayList<MererEntity> bodyDatas) {
        this.bodyDatas = bodyDatas;
    }
}

File: app/src/main/java/com/intfocus/yonghuitest/bean/dashboard/kpi/MeterRequestResult.kt
package com.intfocus.yonghuitest.bean.dashboard.kpi

import com.intfocus.yonghuitest.bean.dashboard.kpi.MererEntity

import java.util.ArrayList

/**
 * 仪表数据请求结果
 * Created by zbaoliang on 17-4-28.
 */
class MeterRequestResult(var isSuccess: Boolean, var stateCode: Int) {
    var topDatas: ArrayList<MererEntity>? = null
    var bodyDatas: ArrayList<MererEntity>? = null
}
